packages/cardhost/tests/integration/components/code-editor-test.js

```diff
@@ -1,18 +1,23 @@
-import { module, skip } from 'qunit';
+import { module, test } from 'qunit';
 import { setupRenderingTest } from 'ember-qunit';
-import { render, waitFor } from '@ember/test-helpers';
+import { render, waitUntil } from '@ember/test-helpers';
 import hbs from 'htmlbars-inline-precompile';
 
 module('Integration | Component | code-editor', function(hooks) {
   setupRenderingTest(hooks);
 
-  skip('it renders', async function(assert) {
-    // Set any properties with this.set('myProperty', 'value');
-    // Handle any actions with this.set('myAction', function(val) { ... });
-
-    await render(hbs`<CodeEditor />`);
-    waitFor('.monaco-editor')
-    assert.equal(this.element.textContent.trim(), '1');
+  test('it renders', async function(assert) {
+    this.set('editorIsReady', false)
+    let editorIsReady = false
+    this.set('editorReady', () => {
+      editorIsReady = true;
+    })
 
+    await render(hbs`<CodeEditor @editorReady={{action this.editorReady}} @code="card" />`);
+    await waitUntil(() => {
+      return editorIsReady;
+    }, {timeout: 10000})
```

Comment:
This is good for now. Eventually we should abstract this into a test helper so that we can more easily test other editor features later.